crypto/txt_db/txt_db.c

```diff
@@ -21,9 +21,8 @@ TXT_DB *TXT_DB_read(BIO *in, int num)
 {
     TXT_DB *ret = NULL;
     int esc = 0;
-    int i, add, n;
-    int size = BUFSIZE;
-    int offset = 0;
+    int n;
+    size_t i, add, size = BUFSIZE, offset = 0;
```

Comment:
You need to add a `size_t` variable that gets initialized from `num`, and use that variable instead of `num` in the for-loop with the current condition `i < num`

Reply:
Fixed :)